pkg/thanos/statefulset.go

```diff
@@ -440,6 +440,7 @@ func makeStatefulSetSpec(tr *monitoringv1.ThanosRuler, config Config, ruleConfig
 	return &appsv1.StatefulSetSpec{
 		Replicas:            tr.Spec.Replicas,
 		MinReadySeconds:     minReadySeconds,
+		ServiceName: governingServiceName,
```

Comment:
you need to run `make format`

Reply:
@simonpasquier, Thanks for your suggestion!